### PR TITLE
Added sortierignore file support

### PR DESCRIPTION
**Pull request by k2snowman69**

### CODE REVIEW

**Comment by k2snowman69** [comment]
@TikiTDO Mind pulling this request and giving it a try? You should be able to npm link to this repository and test if the sortierignore file works for you

File: src/reprinter/index.ts [hunk 1]
```diff
@@ -14,6 +16,19 @@ export class Reprinter {
     new JsonReprinter()
   ];
   public static rewriteFile(filename: string, options: ReprinterOptions) {
+    const ignoreFilePath = path.resolve(".sortierignore");
+    try {
+      let ignoreText = FileUtils.readFileContents(ignoreFilePath).trim();
+      const relativeFilePath = path.relative(path.resolve("."), filename);
+      if (0 < ignoreText.length) {
+        let ig = ignore();
+        ig.add(ignoreText.split(/\r?\n/));
```
Comment (TikiTDO) [inline src/reprinter/index.ts]: Could also filter out comments here:

```suggestion
        ig.add(ignoreText.split(/\r?\n/).filter((word) => !word.match(/^\s*#/));
```

Comment (k2snowman69) [reply src/reprinter/index.ts]: Comments are already handled by the ignore package
https://www.npmjs.com/package/ignore#addpatterns-arraystring--ignore-this

**Comment by k2snowman69** [comment]
Ran unit tests on osx and windows, seems to be good so far

File: src/reprinter/index.ts (hunk 1 above)
Comment (TikiTDO) [reply src/reprinter/index.ts]: Ah, nice

**Pull request merged**